contracts/PositionManager.sol

```diff
@@ -157,42 +181,32 @@ contract PositionManager is IPositionManager, ERC721Holder {
         emit ModuleStateChanged(moduleAddress, tokenId, activated);
     }
 
-    ///@notice return the state of the module for tokenId position
-    ///@param tokenId ID of the position
-    ///@param moduleAddress address of the module
-    function getModuleState(uint256 tokenId, address moduleAddress)
-        external
-        view
-        override
-        onlyOwnedPosition(tokenId)
-        returns (bool)
-    {
-        return activatedModules[tokenId][moduleAddress].isActive;
-    }
-
     ///@notice sets the data of a module strategy for tokenId position
     ///@param tokenId ID of the position
     ///@param moduleAddress address of the module
     ///@param data data for the module
     function setModuleData(
         uint256 tokenId,
         address moduleAddress,
-        bytes memory data
+        bytes32 data
     ) external override onlyOwner onlyOwnedPosition(tokenId) {
+        uint256 moduleData = uint256(data);
+        require(moduleData > 0, 'PositionManager::setModuleData: moduleData must be greater than 2%');
         activatedModules[tokenId][moduleAddress].data = data;
     }
 
-    ///@notice returns the data of a module strategy for tokenId position
-    ///@param tokenId ID of the position
-    ///@param moduleAddress address of the module
-    function getModuleData(uint256 tokenId, address moduleAddress)
+    ///@notice get info for a module strategy for tokenId position
+    ///@param _tokenId ID of the position
+    ///@param _moduleAddress address of the module
+    ///@return isActive is module activated
+    ///@return data of the module
+    function getModuleInfo(uint256 _tokenId, address _moduleAddress)
```

Comment:
What is the advantage of calling getModuleInfo(id, address) vs activatedModules(id, address)?

Reply:
you cant call mapping externally using interface, if you want to call it you need to import directly the contract, waste of gas